addons/website/views/snippets/s_card.xml

```diff
@@ -68,51 +71,43 @@
                 <we-button data-select-class="ratio ratio-4x3" data-dependencies="card_img_top_opt">Landscape - 4/3</we-button>
                 <we-button data-select-class="ratio ratio-16x9" data-dependencies="card_img_top_opt">Wide - 16/9</we-button>
                 <we-button data-select-class="ratio ratio-21x9" data-dependencies="card_img_top_opt">Ultrawide - 21/9</we-button>
-                <we-button data-select-class="ratio o_card_img_ratio_custom" data-dependencies="card_img_top_opt">Custom</we-button>
+                <we-button data-select-class="ratio o_card_img_ratio_custom" data-dependencies="card_img_top_opt" data-name="image_ratio_custom_opt">Custom</we-button>
             </we-select>
 
             <!-- Get bootstrap default column size -->
             <t t-set="colSize" t-value="100/12"/>
-            <we-range string="Width"
+            <we-range string="Width" class="o_we_sublevel_1"
                 title="Adjust the image width"
+                data-name="cover_image_width_opt"
                 data-dependencies="!card_img_top_opt"
                 data-select-style="0"
                 data-css-property="--card-img-size-h"
                 data-unit="%"
                 t-att-data-min="colSize"
                 t-att-data-max="colSize * 11"
-                t-att-data-step="colSize"
-                class="o_we_sublevel_1"/>
+                t-att-data-step="colSize"/>
```

Comment:
On a second thought, I think QSM does not like when we use `calc` :thinking: 
So instead what you can do is simply replace
```xml 
<t t-set="colSize" t-value="100/12"/>
```
by the bootstrap value 
```xml
<t t-set="colSize" t-value="8.33333333"/>
```
=> it will avoid having the rounding errors :slightly_smiling_face: 

And in the SCSS, replace the value of `--card-img-size-h` by `33.33333333%` (given what I wrote just above) :wink: